### PR TITLE
Update code samples

### PR DESCRIPTION
**Pull request by Ndiritu**
closes https://github.com/microsoftgraph/msgraph-sdk-php/issues/1370
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoftgraph/msgraph-sdk-php/pull/1388)

### CODE REVIEW

File: docs/Examples.md [hunk 1]
```diff
@@ -312,16 +320,29 @@ $attachmentItem->setSize($file->getSize());
...
+            if ($uploadSession) {
+                break;
+            }
+        } catch (NetworkExceptionInterface $ex) {
+            $retries ++;
+        }
+    }
+    if ($retries >= $maxRetries && !$uploadSession) {
+        $cancelledUploadSession = $largeFileUpload->cancel();
+    }
```
Comment (Ndiritu) [inline docs/Examples.md:345]: @waldekmastykarz thoughts?

File: docs/Examples.md [hunk 2]
```diff
@@ -312,16 +320,27 @@ $attachmentItem->setSize($file->getSize());
...
 // upload
 $largeFileUpload = new LargeFileUploadTask($uploadSession, $graphServiceClient->getRequestAdapter(), $file);
-try{
+try {
     $uploadSession = $largeFileUpload->upload()->wait();
-} catch (\Psr\Http\Client\NetworkExceptionInterface $ex) {
+} catch (NetworkExceptionInterface $ex) {
     // resume upload in case of network errors
-    $uploadSession = $largeFileUpload->resume()->wait();
+    $retries = 0;
```
Comment (Ndiritu) [inline docs/Examples.md:331]: @waldekmastykarz thoughts?

File: docs/Examples.md [hunk 3]
```diff
@@ -312,16 +320,27 @@ $attachmentItem->setSize($file->getSize());
...
+        try {
+            $uploadSession = $largeFileUpload->resume()->wait();
+            if ($uploadSession) {
+                break;
+            }
+        } catch (NetworkExceptionInterface $ex) {
+            $retries ++;
+        }
+    }
+    throw $ex;
```
Comment (Ndiritu) [inline docs/Examples.md:343]: @waldekmastykarz thoughts?

Comment (waldekmastykarz) [reply docs/Examples.md:343]: Nice!

**Review by isvargasmsft — APPROVED** [review]

**Pull request merged**